apps/dcos_overlay/src/dcos_overlay_poller.erl

```diff
@@ -339,6 +501,36 @@ parse_vtep_mac(MAC) ->
         end,
         MACComponents).
 
+%%%===================================================================
+%%% Metrics functions
+%%%===================================================================
+
+-spec(init_metrics() -> ok).
+init_metrics() ->
+    prometheus_counter:new([
```

Comment:
ditto: we don't need it anymore, there's `poll_duration_seconds_cnt`

Reply:
Please refer to my response regarding `configuration_update_processing_duration_seconds_cnt`